app/src/main/java/com/macoscope/ketchuplunch/presenter/LoginPresenter.kt

```diff
@@ -20,9 +20,9 @@ class LoginPresenter(val loginView: LoginView,
                      val googleCredentialsWrapper: GoogleCredentialWrapper) {
 
 
-    private val REQUEST_ACCOUNT_PICKER = 1000
-    private val REQUEST_AUTHORIZATION = 1001
-    private val REQUEST_GOOGLE_PLAY_SERVICES = 1002
+    private var REQUEST_ACCOUNT_PICKER = 1000
+    private var REQUEST_AUTHORIZATION = 1001
+    private var REQUEST_GOOGLE_PLAY_SERVICES = 1002
```

Comment:
Agree, those variables should stay read-only, hence `val` keyword should be used